controllers/flowlogspipeline/flp_common_objects.go

```diff
@@ -319,6 +319,89 @@ func (b *builder) addTransformStages(stage *config.PipelineBuilderStage) (*corev
 		// Else: nothing for eBPF at the moment
 	}
 
+	outputFields := []api.OutputField{
+		{
+			Name:      "Bytes",
+			Operation: "sum",
+		},
+		{
+			Name:      "Bytes",
+			Operation: "sum",
+			SplitAB:   true,
+		},
+		{
+			Name:      "Packets",
+			Operation: "sum",
+		},
+		{
+			Name:      "Packets",
+			Operation: "sum",
+			SplitAB:   true,
+		},
+		{
+			Name:      "numFlowLogs",
+			Operation: "count",
+		},
+		{
+			Name:      "TimeFlowStartMs",
+			Operation: "min",
+		},
+		{
+			Name:      "TimeFlowEndMs",
+			Operation: "max",
+		},
+		{
+			Name:      "FlowDirection",
+			Operation: "first",
+		},
+		{
+			Name:      "IfDirection",
+			Operation: "first",
+		},
+		{
+			Name:      "AgentIP",
+			Operation: "first",
+		},
+	}
+
+	if helper.IsTCPDropEnabled(b.desired) {
+		outputTCPDropFields := []api.OutputField{
+			{
+				Name:      "TCPDropBytes",
+				Operation: "sum",
+			},
+			{
+				Name:      "TCPDropBytes",
+				Operation: "sum",
+				SplitAB:   true,
+			},
+			{
+				Name:      "TCPDropPackets",
+				Operation: "sum",
+			},
+			{
+				Name:      "TCPDropPackets",
+				Operation: "sum",
+				SplitAB:   true,
+			},
+		}
+		outputFields = append(outputFields, outputTCPDropFields...)
+	}
+
+	if helper.IsDNSTrackingEnabled(b.desired) {
+		outDNSTrackingFields := []api.OutputField{
+			{
+				Name:      "DnsRequestTimeMs",
+				Operation: "min",
+			},
+			{
+				Name:      "DnsResponseTimeMs",
+				Operation: "max",
+			},
+		}
```

Comment:
I can see request and response times using this method however we don't have both in the same conversation since we [reintroduced FlowDirection](https://issues.redhat.com/browse/NETOBSERV-915) to avoid duplicates.

![image](https://github.com/netobserv/network-observability-operator/assets/91894519/297af7c0-bff5-4b86-8971-b8c84fab55e8)

Adding DNS Id on conversations may also be tricky since it rely on `min` / `max` aggregators on `DnsRequestTimeMs` / `DnsResponseTimeMs` fields.